### ISSUE TITLE
add more tensor flow graphs

### ISSUE THREAD

**Issue by ChristofHenkel** [ChristofHenkel/speech_recognition#11]
e.g.

- gradient norm
- clipped gradient norm
- learning rate
- trainingtime/ batch

**Comment by ChristofHenkel** [comment]
done
